src/main/java/io/buildpacks/example/sample/GreetingController.java

```diff
@@ -7,8 +7,14 @@
 @RestController
 public class GreetingController {
 
-    @GetMapping("/greet")
-    public String greet(@RequestParam(value = "name", defaultValue = "World") String name) {
```

Comment:
5. Security:
   - Ensure that proper security measures are in place to handle unexpected exceptions in production to prevent leaking sensitive information.